test/models/flava/test_flava.py

```diff
@@ -72,18 +99,19 @@ def test_forward_pretraining(self):
             itm_labels=itm_labels,
             mlm_labels=mlm_labels,
         )
-        self.assertIsNone(output.mlm_output)
-        self.assertIsNone(output.mim_output)
-        self.assertIsNotNone(output.global_contrastive_output)
-        self.assertIsNotNone(output.mmm_text_output)
-        self.assertIsNotNone(output.mmm_image_output)
-        self.assertIsNotNone(output.itm_output)
-        self.assertAlmostEqual(
+        assert output.mlm_output is None
```

Comment:
```
expected = {
    "mlm_output": None,
    "mim_output": None,
    "global_constrastive_output": (size, sum),
    "losses": (size, 21.5150),
}
assert_expected_namedtuple(actual, expected, rtol=1e-5, atol=1e-4)
```

Reply:
Ultimately decided to leave this as is, mainly because the test is currently not explicitly checking sizes and values of all these components and the final loss value check should be sufficient here